app/controllers/api/v1/slack_users_controller.rb

```diff
@@ -28,6 +50,24 @@ def authenticate_py_bot!
       def auth_token
         @auth_token ||= request.headers.fetch('Authorization', '').split(' ').last
       end
+
+      def generate_client_action(user, email)
```

Comment:
@wimo7083 This is what I did for the 3d Items on the issue: "sends new POST to controlling endpoint for slack telling to send email update request."

Just return a action value that details if no further action needs to happen or if the client needs to request a email update to the user table